src/components/DashboardImmersion/ImmersionCrudForm.js

```diff
@@ -11,12 +11,14 @@ import { definitions } from 'common/utils/validationHelpers'
 
 function ImmersionCrudForm({ dataToEdit, site, submitHandler }) {
   const validator = yup.object().shape({
-    dictionaryEntry: yup.object().required('A label is required'),
+    dictionaryEntry: definitions
+      .objectArray()
+      .min(1, 'An entry is required for this label.'),
```

Comment:
Is this going to prevent people from removing an dictionary entry link if they make a mistake?

Reply:
Or are all of them required in order to save the set? I'm not really familiar with how it works

Reply:
Not all of them are required, the labels save individually, however; currently the BE doesn't allow the setting of the immersion label dictionary_entry property to blank/null. We could delete the label if a user removes the link? I will merge this for now and we can consider if we want to change the functionality at a later date.